src/components/DetailItem/DetailItem.tsx

```diff
@@ -1,7 +1,9 @@
-export interface DetailItemType {
+import classNames from 'classnames';
+import { WithClassnameType } from 'types';
+
+export interface DetailItemUIType extends WithClassnameType {
   children?: React.ReactNode;
```

Comment:
Here, `ReactNode` could be directly imported from React.

Reply:
updated

Reply:
Great. 🔥